src/components/ChatList/GroupChatChannelItem.tsx

```diff
@@ -20,23 +21,41 @@ const GroupChatChannelItem = (props: ChannelItemProps) => {
   const unreadCount = groupChat?.unreadCount;
   const hasBadge = unreadCount > 0;
 
-  const description = groupChat?.rawJson?.firstMessage ?? groupChat?.rawJson?.message;
-  const isSystemDescription =
-    description?.isSystem ||
-    description?.type === 'system' ||
-    description?.message?.type === 'system';
-  const isDeletedMessage = description?.type === 'deleted';
-  const sender = description?.user?.username ?? description?.user?.name;
-  const isMeAsSender =
-    description?.user?.id === signedProfileId || description?.user?.isMe || !description;
+  const isMe = groupChat?.lastUpdatedBy === signedProfileId;
+  const firstMessage = groupChat?.firstMessage;
+  const firstMessageText = firstMessage?.message;
+  let description = groupChat?.description;
 
-  let channelDescription = groupChat?.description;
-  if (!isSystemDescription && description !== channelDescription) {
-    channelDescription = isMeAsSender
-      ? `You: ${channelDescription}`
-      : `${sender}: ${channelDescription}`;
+  if (['deleted', 'notification-deleted'].includes(groupChat?.firstMessage?.type || '')) {
+    description = DELETED_MESSAGE_TEXT;
+  } else if (groupChat?.firstMessage?.type === 'system') {
+    description = firstMessageText || '';
+  } else if (isMe) {
+    description = `You: ${description}`;
+  } else if (!isMe) {
+    description = `${groupChat?.user?.username}: ${description}`;
   }
-  if (isDeletedMessage) channelDescription = '';
+
+  const getMessageText = (): React.ReactNode => {
+    let attachmentJson;
+
+    try {
+      attachmentJson = JSON.parse(groupChat?.firstMessage?.attachmentJson || '[]');
+    } catch (e) {
+      console.log('error parsing attachment json', e);
+    }
+
+    if (attachmentJson?.length > 1) {
+      return (
+        <Text>
+          <Text>{isMe ? 'You: ' : `${groupChat?.user?.username}: `}</Text>
+          <Text style={{fontStyle: 'italic'}}>Sent media 🏞️</Text>
+        </Text>
+      );
+    }
+
+    return `${description}`;
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for constructing and displaying channel description has been refactored. However, there is a potential issue with error handling. When parsing the `attachmentJson`, if an error occurs, it's simply logged to the console and the function continues execution. This could lead to unexpected behavior if `attachmentJson` is expected to be an array later in the code. Consider throwing the error or handling it in a way that doesn't allow the function to continue executing with potentially invalid data.

```diff
    try {
      attachmentJson = JSON.parse(groupChat?.firstMessage?.attachmentJson || '[]');
    } catch (e) {
      console.log('error parsing attachment json', e);
+       throw e;
    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->